web/themes/custom/cloudy/templates/field/field--field-end-time.html.twig

```diff
@@ -0,0 +1,17 @@
+{% embed 'field.html.twig' %}
+  {% block content %}
+    {% if multiple %}
+      <div class="field__items">
+    {% endif %}
+    {% for item in items %}
+      <div{{ item.attributes.addClass(item_classes) }}>
+        {{ icon }}
```

Comment:
The End Time field shouldn't include an icon.

Reply:
Got it.